docker-compose.yml

```diff
@@ -43,6 +43,7 @@ services:
       FN_DB_URL: "mysql://root:root@tcp(db:3306)/funcs"
       FN_MQ_URL: "redis://mq:6379/"
       FN_LOGSTORE_URL: "s3://admin:password@logstore:9000/us-east-1/fnlogs"
+      FN_DOCKER_NETWORKS: "fn_default"
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
   grafana:
```

Comment:
you need to add network definition:
```yaml
networks:
  fn-network:
   name: fn-network
```
and then add each service (datastore, log store, fn, fn ui) to that network:
```yaml
    networks:
      - fn-network
```
then set FN_DOCKER_NETWORKS:
```yaml
FN_DOCKER_NETWORKS: "fn-network"
```
This needs to be done explicitly.

Reply:
done.